chapters/functions.tex

```diff
@@ -2156,26 +2142,25 @@ \subsection{Annotations for External Libraries and Include Files}\doublelabel{an
   specified by using an URI name for the library directory, see \autoref{external-resources}.
   Different versions of one object library can be provided
   \emph{{[}e.g. for Windows and for Linux{]}} by providing a
-  ``platform'' directory below the ``LibraryDirectory''. If no
-  ``platform'' directory is present, the object library must be present
-  in the ``LibraryDirectory''. The following ``platform'' names are
+  \lstinline!platform! directory below the \lstinline!LibraryDirectory!. If no
```

Comment:
I don't think that platform should be seen as code, as it is seen as the platform-variable.

Reply:
Right.  I was unsure whether any emphasis was needed at all, but ended up with `\emph` around the first and last occurrences.